internal/benchrunner/runners/stream/runner.go

```diff
@@ -120,15 +124,58 @@ func (r *runner) setUp() error {
 	}
 	r.scenarios = scenarios
 
-	if err = r.installPackage(); err != nil {
-		return fmt.Errorf("error installing package: %w", err)
-	}
-
 	err = r.collectGenerators()
 	if err != nil {
 		return fmt.Errorf("can't initialize generator: %w", err)
 	}
 
+	return nil
+}
+
+const NumberOfEvents = 10
```

Comment:
Nit: It is common to have const on top of the doc. And as it is not used outside the scope (yet), it can start with lowercase.